BeardedSpice/BSVolumeControlProtocol.h

```diff
@@ -0,0 +1,30 @@
+//
+//  BSVolumeControlProtocol.h
+//  BeardedSpice
+//
+//  Created by Roman Sokolov on 15.05.17.
+//  Copyright (c) 2015-2017 GPL v3 http://www.gnu.org/licenses/gpl.html
+//
+
+#ifndef BSVolumeControlProtocol_h
+#define BSVolumeControlProtocol_h
+
+typedef enum {
```

Comment:
Please use the objc convention `BSVolumeControl-Protocol.h` for the filename.

Please use the apple helper `typedef NS_ENUM(NSInteger, BSVolumeControlResult) {}` for all enums.
http://nshipster.com/ns_enum-ns_options/